Sources/JavaKit/JavaKitVM/JavaVirtualMachine.swift

```diff
@@ -54,6 +60,12 @@ public final class JavaVirtualMachine: @unchecked Sendable {
     // Construct the complete list of VM options.
     var allVMOptions: [String] = []
     if !classPath.isEmpty {
+      let fileManager = FileManager.default
+      for path in classPath {
+        if !fileManager.fileExists(atPath: path) {
+          throw JavaKitError.classPathNotFound(path)
```

Comment:
```suggestion
          throw JavaKitError.classPathSegmentNotFound(path)
```

It's not that the classpath isn't found, just that one of its segments/paths isnt, shall we rename it like this?